src/process_forest.py

```diff
@@ -449,6 +465,7 @@ def main():
     serialize_parser.add_argument("pt", type=str, default="state.pt",
                         help=".pt file to serialize parsed trees")
 
+    global args
```

Comment:
I'm not too keen on using a global variable here. Would you be willing to update the PR to thread a `pid_formatter` argument to the relevant calls?

might look something like:

```
if args.hexpids:
    pid_formatter = hex
else:
    pid_formatter = str

analyzer = ProcessTreeAnalyzer(pid_formatter=pid_formatter)
```